cluster/addons/device-plugins/nvidia-gpu/daemonset.yaml

```diff
@@ -44,10 +44,10 @@ spec:
         resources:
           requests:
             cpu: 50m
-            memory: 10Mi
+            memory: 50Mi
```

Comment:
This is a substantial increase that will cause eviction of user workload on upgrade.